packages/flutter/lib/src/rendering/mouse_tracking.dart

```diff
@@ -351,7 +350,7 @@ abstract class BaseMouseTracker extends ChangeNotifier {
         as LinkedHashMap<MouseTrackerAnnotation, Matrix4>;
     for (final HitTestEntry entry in result.path) {
       if (entry.target is MouseTrackerAnnotation) {
-        annotations[entry.target as MouseTrackerAnnotation] = entry.transform;
+        annotations[entry.target as MouseTrackerAnnotation] = entry.transform!;
```

Comment:
is transform guaranteed to be non-null?

Reply:
It shouldn't be null once added to path.

```dart
  /// Add a [HitTestEntry] to the path.
  ///
  /// The new entry is added at the end of the path, which means entries should
  /// be added in order from most specific to least specific, typically during an
  /// upward walk of the tree being hit tested.
  void add(HitTestEntry entry) {
    assert(entry._transform == null);
    entry._transform = _lastTransform;
    _path.add(entry);
  }
```